infra/repository/locals.tf

```diff
@@ -4,8 +4,9 @@ locals {
   identity_resource_group_name = "${local.project}-identity-rg"
 
   repo_secrets = {
-    "ARM_TENANT_ID"       = data.azurerm_client_config.current.tenant_id,
-    "ARM_SUBSCRIPTION_ID" = data.azurerm_subscription.current.subscription_id
+    "ARM_TENANT_ID"           = data.azurerm_client_config.current.tenant_id,
+    "ARM_SUBSCRIPTION_ID"     = data.azurerm_subscription.current.subscription_id
+    "ARM_DEV_SUBSCRIPTION_ID" = data.azurerm_subscription.dev.subscription_id
```

Comment:
we need another identity and another github environment to manage DEV-IO as current identities have roles on PROD-IO only

Reply:
why `dx-typescript` runners have roles on PROD-IO ?

Reply:
we have moved the state into `DEV-ENGINEERING` subscription